.github/workflows/smoke-checks.yml

```diff
@@ -22,68 +22,67 @@ concurrency:
 
 env:
   HOMEBREW_NO_INSTALL_CLEANUP: 1 # Disable cleanup for homebrew, we don't need it on CI
-  XCODE_VERSION: 13.4.1
-  IOS_VERSION: "15.5"
-  IOS_SIMULATOR_DEVICE: "iPhone 12 Pro"
-  IOS_SIMULATOR_DEVICE_WITH_VERSION: "iPhone 12 Pro (15.5)"
+  XCODE_VERSION: "14.0.1"
+  IOS_VERSION: "15.4"
+  IOS_SIMULATOR_DEVICE_WITH_VERSION: "iPhone 12 Pro (15.4)"
```

Comment:
Why did we downgrade here?

Reply:
Egh, I did not find a way how to install `15.5` on Xcode 14. We only have `16` and `15.4` out there. As snapshot tests do not work well enough on iOS `16`, the only option is `15.4`. See [failed job](https://github.com/GetStream/stream-chat-swift/actions/runs/3225236953/jobs/5277303724) and screenshot attached:
<img width="837" alt="Screen Shot 2022-10-11 at 9 24 24 AM" src="https://user-images.githubusercontent.com/20794991/195039228-63228b0f-7579-42b3-a021-52e04f5ce0fa.png">